src/schemas/channels.ts

```diff
@@ -0,0 +1,7 @@
+export interface ChannelsPayload {
```

Comment:
Could you please add a comment in detail for introducing what is a Channel Message?

It would be great to provide some links to related Channel Message articles/blog posts/news etc.

Reply:
OK, add a [link](https://wechatwiki.com/wechat-resources/wechat-channels-short-video-feature-complete-guide/) for it.